{"name": "test_9", "status": "failed", "statusDetails": {"message": "AssertionError: assert 3 == 4", "trace": "@allure.feature(\"模块三\")\n    @allure.severity(\"trivial\")\n    def test_9():\n>       assert 3 == 4\nE       assert 3 == 4\n\ntest_1.py:76: AssertionError"}, "start": 1566182588869, "stop": 1566182588870, "uuid": "4e35c415-ea14-4eea-9ea3-14d9166fe4b9", "historyId": "4bfd404cc156e55d67f452d47146152a", "fullName": "test_1#test_9", "labels": [{"name": "feature", "value": "模块三"}, {"name": "severity", "value": "trivial"}, {"name": "parentSuite"}, {"name": "suite", "value": "test_1"}, {"name": "subSuite"}, {"name": "host", "value": "pc-guohonggang"}, {"name": "thread", "value": "8808-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_1"}]}